src/server/services/local_meta_service.h

```diff
@@ -50,6 +50,18 @@ class LocalMetaService : public IMetaService {
 
   ~LocalMetaService() override {}
 
+  void TryAcquireLock(std::string key,
+                      callback_t<bool, std::string> callback_after_try_locked) {
+    // TBD
+    assert(false);
+  }
+
+  void TryReleaseLock(std::string key,
+                      callback_t<bool> callback_after_try_unlocked) {
+    // TBD
+    assert(false);
+  }
```

Comment:
?

Reply:
I have change it to Status::NotImplemented